tests/conftest.py

```diff
@@ -1,30 +1,15 @@
 from pathlib import Path
 import numpy as np
 import scanpy as sc
+import torch
 import pytest
 from paste3.helper import intersect
-import torch
 import ot.backend
 
 test_dir = Path(__file__).parent
 input_dir = test_dir / "data/input"
 
 
-def pytest_generate_tests(metafunc):
```

Comment:
After we modified test to check on fewer combination of parameters, the remaining parametrization only happens on a single combination of parameters, so we don't need to parametrize anyway.